docs/user/convert/parsing_and_validation.rst

```diff
@@ -117,7 +166,7 @@ incremental parse mode:
 
     geomesa.converters.myconverter {
       options {
-        validators = [ "z-index" ]
+        validators = [ "index" ]
```

Comment:
Since there's a string change, do we support configs with 'z-index'?  Or will old converters die?

Reply:
it's still supported, looks like you found it already